utils/config.ts

```diff
@@ -5,35 +5,54 @@
  * The configuration is evaluate eagerly at the first access to the module. The module exposes convenient methods to access such value.
  */
 
+import { fromNullable } from "fp-ts/lib/Option";
 import * as t from "io-ts";
 import { readableReport } from "italia-ts-commons/lib/reporters";
 import { NonEmptyString } from "italia-ts-commons/lib/strings";
 
+export const RedisParams = t.intersection([
+  t.interface({
+    REDIS_URL: NonEmptyString
+  }),
+  t.partial({
+    REDIS_CLUSTER_ENABLED: t.boolean,
+    REDIS_PASSWORD: NonEmptyString,
+    REDIS_PORT: NonEmptyString
```

Comment:
Could you add this ENVs inside the `env.example` file? We should upgrade even `io-mock` configuration for `fn-cgn`